taskcat/stacker.py

```diff
@@ -310,12 +310,7 @@ def get_param_includes(self, original_keys):
         except Exception as e:
             pass
 
-        # Setup a list index dictionary.
```

Comment:
refactored to it's own function to improve readability/testability and contribute towards reducing sprawl in `stacker.py`